dbms/dbms.c

```diff
@@ -0,0 +1,175 @@
+#include "dbms.h"
+#include "../ipc/messages.h"
+#include "../types/types.h"
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <sys/ipc.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+
+void test_put_vec_len_1(int queue_id, int vec_id, unsigned long long vec) {
+    vec_t *v = (vec_t *)malloc(sizeof(vec_t));
+	unsigned long long *vector = (unsigned long long *)malloc(sizeof(unsigned long long) * MAX_VECTOR_LEN);
+	//vector[0] = vec;
+	unsigned long long varr[] = {vec};
```

Comment:
We are getting the following warning:
```
dbms/dbms.c:20:22: warning: unused variable 'vector'
```
I'm guessing `varr[]` replaced `*vector` so we could probably remove lines 20–21.